docs/developer/plugin-api-changes/plugin-api-changes.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[plugin-api-changes]]
+== Plugin API changes in {minor-version}
+++++
+<titleabbrev>Plugin API changes</titleabbrev>
+++++
+
+This page discusses the plugin API changes that you need to be aware of when migrating
+your application to {kib} {minor-version}.
+
+Other versions: {kibana-ref-all}/7.17/plugin-api-changes.html[7.17] |
```

Comment:
In the release notes, I removed the 7.x branches since this doc is 8.0-specific.